Formats/Legacy/metas.json

```diff
@@ -2,12 +2,20 @@
    Metas:
    [
       { "StartDate": "2020-01-17", "Name": "PostTherosBeyondDeath" },
+      { "StartDate": "2020-03-10", "Name": "PostBreachBan" },
       { "StartDate": "2020-04-17", "Name": "PostIkoria" },
+      { "StartDate": "2020-05-18", "Name": "PostLurrusBan" },
```

Comment:
2020-05-19
https://magic.wizards.com/en/articles/archive/news/may-18-2020-banned-and-restricted-announcement
The day after the announcement